hack/generate-swagger-api.sh

```diff
@@ -83,6 +83,9 @@ EOT
 
 generate_model types/storage <<- 'EOT'
 	DriverData
+	Storage
+	RootFSStorage
+	RootFSStorageSnapshot
```

Comment:
😅 https://github.com/moby/moby/blob/8025c404936b901765628513bf857d8319f1e71f/hack/generate-swagger-api.sh#L28-L36


```suggestion
	RootFSStorage
	RootFSStorageSnapshot
	Storage
```